reana/cli.py

```diff
@@ -1836,6 +2138,182 @@ def run_example(
     run_command("echo OK", component)
 
 
+@click.option(
+    "-m",
+    "--mount",
+    "mounts",
+    multiple=True,
+    callback=volume_mounts_to_list,
+    help="Which local directories to mount in the cluster nodes? [local_path:cluster_node_path]",
+)
+@click.option(
+    "--mode",
+    default="latest",
+    help="In which mode to run REANA cluster? (releasehelm,releasepypi,latest,debug) [default=latest]",
+)
+@click.option("--worker-nodes", default=0, help="How many worker nodes? [default=0]")
+@cli.command(name="cluster-create")
+def cluster_create(mounts, mode, worker_nodes):  # noqa: D301
+    """Create cluster.
+
+    \b
+    Example:
+       $ reana-dev cluster-create -m /var/reana:/var/reana
+                                  -m /usr/share/local/mydata:/mydata
+                                  --mode debug
+    """
+    import sys
+
+    class literal_str(str):
+        pass
+
+    def literal_unicode_str(dumper, data):
+        return dumper.represent_scalar("tag:yaml.org,2002:str", data, style="|")
+
+    def add_volume_mounts(node):
+        """Add needed volumes mounts to the provided node."""
+
+    yaml.add_representer(literal_str, literal_unicode_str)
+
+    control_plane = {
+        "role": "control-plane",
+        "kubeadmConfigPatches": [
+            literal_str(
+                "kind: InitConfiguration\n"
+                "nodeRegistration:\n"
+                "  kubeletExtraArgs:\n"
+                '    node-labels: "ingress-ready=true"\n'
+            )
+        ],
+        "extraPortMappings": [
+            {"containerPort": 30080, "hostPort": 30080, "protocol": "TCP"},
+            {"containerPort": 30443, "hostPort": 30443, "protocol": "TCP"},
+        ],
+    }
+
+    if mode in ("debug"):
+        mounts.append({"hostPath": find_reana_srcdir(), "containerPath": "/code"})
+        control_plane["extraPortMappings"].extend(
+            [
+                {"containerPort": 31984, "hostPort": 31984, "protocol": "TCP"},
+                {"containerPort": 32580, "hostPort": 32580, "protocol": "TCP"},
```

Comment:
What do you think about having some comments here saying that these ports belong to wdb and maildev?